restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -126,6 +126,11 @@ public Response postDoc(
       @ApiParam(value = "the name of the collection", required = true) @PathParam("collection-id")
           String collection,
       @ApiParam(value = "The JSON document", required = true) String payload,
+      @ApiParam(
+              value = "Include this to put a time-to-live on the document in the data store",
+              required = false)
+          @QueryParam("ttl")
+          Integer ttl,
```

Comment:
Validation that is bigger than 0? This way I can pass negative values..

Reply:
What does the error look like when using an annotation? Is it consistent with our other 400s?

Reply:
Yes there is peace of code that takes that message from annotation and forms error response in the common form -> [JerseyViolationExceptionMapper](https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/impl/JerseyViolationExceptionMapper.java)..